runtime/src/iree/builtins/ukernel/common.h

```diff
@@ -428,6 +428,20 @@ static inline iree_uk_type_t iree_uk_unpack_type(int pos,
 #define IREE_UK_ASSUME_UNREACHABLE
 #endif
 
+// Queries for [[attribute]] identifiers in modern compilers.
+#ifdef __has_attribute
+#define IREE_UK_HAVE_ATTRIBUTE(x) __has_attribute(x)
```

Comment:
`IREE_UK_HAS_ATTRIBUTE`, for consistency with `__has_attribute` and the above `IREE_UK_HAS_BUILTIN` ?

Reply:
for consistency this is following the iree/base/attributes.h for consistency, which came from absl/google3: https://github.com/abseil/abseil-cpp/blob/master/absl/base/attributes.h#L50